ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -60,22 +60,21 @@ export const defaultCromwellConfig: CreateAppRequest = {
   },
 };
 
-// visible/actionable logic is copied from runtime-utils
-
 export const isVisible = (status: AppStatus): boolean =>
-  status && ![AppStatus.DELETED, AppStatus.ERROR].includes(status);
+  status && ![AppStatus.DELETED].includes(status);
 
 export const shouldShowApp = (app: UserAppEnvironment): boolean =>
   isVisible(app?.status);
 
+// TODO what about ERROR?
 export const canCreateApp = (app: UserAppEnvironment): boolean =>
   !isVisible(app?.status);
 
-export const isActionable = (status: AppStatus): boolean =>
-  [AppStatus.RUNNING, AppStatus.STOPPED].includes(status);
+export const isDeletable = (status: AppStatus): boolean =>
+  [AppStatus.RUNNING, AppStatus.ERROR].includes(status);
```

Comment:
can delete ERROR state apps, and I also discovered that deleting STOPPED apps cause 409 Conflict.  So: don't let users try to do that.